INSTALL/install.sh

```diff
@@ -124,3 +124,24 @@ mkdir -p /var/log/reform
 cp ./conf/supervisor*ini /etc/supervisord.d/
 systemctl start supervisord
 systemctl enable supervisord
+
+
+# Create the following cron job
+crontab -e
+# add following command to the files, and save it.
+0 1 * * * /bin/bash /home/reform/reformWeb/cleanup.sh
+
+# Create local File folder for test site
+mkdir -p /home/reform/reformWeb/staticData
```

Comment:
Do we add this to .gitignore? Especially when the static files are so large. Probably too large for github. We can investigate using the test data from the reform repo, https://github.com/gencorefacility/reform/tree/master/test_data.